mifosng-android/src/main/java/com/mifos/api/datamanager/DataManagerDocument.java

```diff
@@ -50,10 +50,10 @@ public DataManagerDocument(BaseApiManager baseApiManager) {
      * @param name       Name of Document
      * @param desc       Description of Document
      * @param file       Document File
-     * @return GenericResponse
+     * @return DocumentRelatedResponse
      */
-    public Observable<GenericResponse> createDocument(String entityType, int entityId, String name,
-                                                      String desc, MultipartBody.Part file) {
+    public Observable<DocumentRelatedResponse> createDocument(
```

Comment:
@tarun0 When we know what will be response for many requests the It's better to follow genericity and Please use `GenericResponse`